airbyte-integrations/connectors/destination-elasticsearch-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchStrictEncryptDestinationAcceptanceTest.java

```diff
@@ -81,6 +94,22 @@ protected TestDataComparator getTestDataComparator() {
   @Override
   protected JsonNode getConfig() {
 
+    final JsonNode authConfig = Jsons.jsonNode(Map.of(
+        "method", "basic",
+        "username", "elastic",
+        "password", "s3cret"));
+
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("endpoint", String.format("https://%s:%s", container.getHost(), container.getMappedPort(9200)))
+        .put("authenticationMethod", authConfig)
+        .put("certAsBytes", container.copyFileFromContainer(
+            "/usr/share/elasticsearch/config/certs/http_ca.crt",
+            InputStream::readAllBytes))
+        .build());
+  }
+
+  protected JsonNode getUnsecureConfig() {
```

Comment:
Is this method called anywhere? Cursory search doesn't doesn't show any other place this _might_ get called. If so, can we remove this method?

Reply:
This will be used in my next PR. Initially, I was going to deliver it in one PR. Will remove it from this one. Thanks

Reply:
removed from this PR, thanks